GraphWebApi/Startup.cs

```diff
@@ -57,6 +58,14 @@ public void ConfigureServices(IServiceCollection services)
                            ValidAudience = Configuration["AzureAd:Audience"],
                            ValidIssuer = Configuration["AzureAd:Issuer"]
                        };
+                       option.Events = new JwtBearerEvents()
+                       {
+                           OnAuthenticationFailed = context =>
+                           {
+                               context.NoResult();
+                               return Task.FromResult(0);
+                           }
+                       };
```

Comment:
I remember adding this to my code to get the debugger to run, thanks for helping @omersayshi 

Could you explain what it does?

Reply:
Thank you for asking this because I overlooked making a comment about this.
This part of the code will not be merged in, this is just to allow for local testing with postman.

Reply:
merged into where?  If it's in here it's being merged into your feature branch.  Should it be removed?

Reply:
@omersayshi correct me if i'm wrong but omer meant it will be removed when this feature branch is being merged into interns/dev